game.py

```diff
@@ -89,12 +76,38 @@ def print_word_progress_string(snowman_word, snowman_word_dict):
         output_string += " "
     print(output_string)
 
-def get_word_progress(snowman_word, snowman_word_dict):
-    """
-    This function takes the snowman_word and snowman_word_dict as input.
-    It returns True if all the letters of the word have been guessed, and False otherwise.
-    """
-    for letter in snowman_word:
-        if not snowman_word_dict[letter]:
-            return False
-    return True
\ No newline at end of file
+
+def snowman(snowman_word):
+    r = RandomWord()
+    snowman_word = r.word(
+        word_min_length=SNOWMAN_MIN_WORD_LENGTH, 
+        word_max_length=SNOWMAN_MAX_WORD_LENGTH)
+
+    print(f"debug info: {snowman_word}")
+
+    correct_guesses_list = []
+    wrong_guesses_list = []
+    
+
+    while len(wrong_guesses_list) < SNOWMAN_WRONG_GUESSES:
```

Comment:
👍 I like that you used the length of the actual list of wrong guesses to tell when the loop should end (rather than storing a count in a separate variable).